modules/dockermodelrunner/sdk/client/create.go

```diff
@@ -0,0 +1,42 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+)
+
+// CreateModel creates a model
+func (c *Client) CreateModel(ctx context.Context, fqmn string) ([]byte, error) {
+	var bytes []byte
+
+	payload := fmt.Sprintf(`{"from": "%s"}`, fqmn)
```

Comment:
suggestion: use %q

Reply:
do we need special characters in the model name?